### ISSUE TITLE
Add Malayalam & Tamil layout

### ISSUE THREAD

**Issue by shockergit** [florisboard/florisboard#261]
Please add 
Malayalam
Tamil 
keyboards

**Comment by patrickgold** [comment]
Thanks for your suggestion! I will place those on my todo list!

**Comment by yashpalgoyal1304** [comment]
duplicate of #157 - that issue is earlier. So, this issue can be closed and the input moved over :)

**Comment by patrickgold** [comment]
Good catch thank you!

**Comment by shockergit** [comment]
This is not duplicate this have 3 layout suggestion. The old issue have only Hindi i think you need to close that and open this

**Comment by patrickgold** [comment]
@shockergit That's true, I've removed the Hindi layout from your suggestion list, so this issue focuses on the other two layouts. This way none of the two issues is a duplicate and both suggest different layouts.

**Comment by shockergit** [comment]
![Screenshot_20210423-133644_Microsoft_SwiftKey_Keyboard](https://user-images.githubusercontent.com/63154802/115840363-3ab58d80-a439-11eb-9e14-1b1389330f4a.png)
Please use this layout for malayalam

**Comment by Tarek-Hasan** [comment]
For reference, you can use [Indic-Keyboard](https://gitlab.com/indicproject/indic-keyboard) project. It includes most used Indic languages and keyboard layouts, including Phonetic and Transliteration keyboard. Which will also help solve other Indic keyboard layout related issues like #1885 and #157.

**Comment by gloriouspuffy** [comment]
Also add handwriting support, if possible since both languages have many characters and handwriting is easier and prefered way to type by many
